src/main/java/se/sundsvall/incident/api/IncidentResource.java

```diff
@@ -100,9 +96,8 @@ ResponseEntity<IncidentOepResponse> getStatusForOeP(
 	}
 
 	@Operation(summary = "Get a list of valid statuses")
-	@ApiResponse(
-		responseCode = "200", description = "Successful Operation", useReturnTypeSchema = true)
-	@GetMapping(value = "/statuses", produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
+	@ApiResponse(responseCode = "200", description = "Successful Operation", useReturnTypeSchema = true)
+	@GetMapping(value = "/incident/statuses", produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
```

Comment:
`@GetMapping(value = "/{municipalityId}/incident/statuses"...` ?

Reply:
These statuses is set automatically when an incident is created. There is API support for updating the status but it is never used. Out of 37664 incidents, none of them have a status that isn't set automatically. 

I was planning on making the statuses data-driven but as the status is never changed and only the defaults are used, i see no real point in doing so. 

I could add the path prefix and just not use it, the benefits would be the flexibility to use it at a later stage without making a new major version, thoughts?

Reply:
Yes, flexibility is allways nice.

But another reason for adding this to the path is the possibility for log separation over different municipalities.